autoload/airline/extensions/searchcount.vim

```diff
@@ -14,7 +14,7 @@ endfunction
 
 function! airline#extensions#searchcount#apply(...) abort
   call airline#extensions#append_to_section('y',
-        \ '%{v:hlsearch ? airline#extensions#searchcount#status() : ""}')
+        \ ' %{v:hlsearch ? airline#extensions#searchcount#status() : ""}')
```

Comment:
Hm, actually, I am wondering why the condition for `v:hlsearch` is there. Doesn't that mean, the searchcount stat will only be shown, when `hls` is on. Is that expectation correct? 

second, please use `g:airline_symbols.space` instead of a literal space. Thanks

Reply:
Yeah if not set hls on, will not show.

Reply:
@chrisbra I know why origin code no space there, if chage to this pr, there will be an extra space when you not in search something by `:something`.
But if no space word will no split in section y.
Seems airline will trime the result of function so add space in return of `airline#extensions#searchcount#status()` not works.
So this pr should be closed, and I agree remove `v:hlsearch`.

Reply:
@chrisbra @roachsinai 

If I remove :v:hlsearch, the searchcount count will remain in section_y even if I use :nohlsearch.
v:hlsearch is included to prevent this from happening.

Reply:
@kazukazuinaina okay, but he can set `let g:airline#extensions#searchcount#enabled = 0` to make it. And I wonder is there a way to insert a SPACE in a better war?

Reply:
>  okay, but he can set let g:airline#extensions#searchcount#enabled = 0 to make it.

what do you mean?
It will turn off the searchcount feature completely.
That's not what I'm trying to say.

## not delete v:hlsearch

![demo1](https://user-images.githubusercontent.com/36619465/103459334-f7cc8e80-4d51-11eb-9d49-b835dfd2b336.gif)


## delete v:hlsearch

![demo2](https://user-images.githubusercontent.com/36619465/103459343-0adf5e80-4d52-11eb-9e4e-71269bc9710f.gif)

Please see these gif.

if we delete `v:hlsearch`, After executing `:nohlsearch`, the result continues to remain in section_y.

Reply:
@kazukazuinaina you're right, I misunderstood it. Thanks for your explanation!

Reply:
@kazuoteramoto Ah yes, that makes sense. We might add this to the documentation.

Reply:
@chrisbra https://github.com/vim-airline/vim-airline/blob/4d284a677c5c54a08d2d111e0cbaf915a28eedcc/autoload/airline/extensions/searchcount.vim#L24
do you use extra `/` in  above to show we are search something?